src/GitHubExtension/Strings/en-US/Resources.resw

```diff
@@ -207,15 +207,19 @@
   </data>
   <data name="LoginUI_LoginFailedPage_text1" xml:space="preserve">
     <value>Something went wrong :(</value>
+    <comment>Generic error text shown in LoginUI failure page</comment>
   </data>
   <data name="LoginUI_LoginFailedPage_text2" xml:space="preserve">
     <value>Please consider filing feedback</value>

```

Comment:
Since this is in Extension UI, we would need to protocol launch DevHome to the Feedback page. This needs DevHome-side changes to handle the protocol launch and navigate to the correct page. Will create an issue for this.

Reply:
https://github.com/microsoft/devhomegithubextension/issues/314